.github/workflows/copilot.yml

```diff
@@ -0,0 +1,59 @@
+name: Copilot Workflows
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+        environment:
+          description:  Which AWS Account to use
+          type: choice
+          required: true
+          options:
+          - test
+        # Shared workflow consideration
+        # application:
+        #   description:  Application Name
+        #   type: string/choice
+        # - notification
+        #   required: true
+        init:
+          description: Initialise the application?
+          type: boolean
+          default: false
+        service:
+          description: Service Name
+          type: string
+          required: true
+          default: 'funding-service-design-frontend'
+        port:
+          description: Access port
+          type: string
+          default: '80'
+        type:
+          description: Type of service to deploy
+          type: choice
+          options:
+          - 'Backend Service'
+          - 'Load Balanced Web Service'
+          - 'Request-Driven Web Service'
+          - 'Scheduled Job'
+          - 'Worker Service'
+          default: 'Backend Service'
+
+
+jobs:
+  deployment:
+    concurrency: deploy-${{ inputs.environment || 'test' }} # Forces only one workflow at a time can run on the environment
+    permissions:
+      id-token: write # This is required for requesting the JWT
+      contents: read  # This is required for actions/checkout
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment || 'test' }}
+    steps:
+      - name: Git clone the repository
+        uses: actions/checkout@v3
+
+      - name: Get current date
+        id: currentdatetime
+        run: echo "::set-output name=datetime::$(date +'%Y%m%d%H%M%S')"
```

Comment:
Nothing to do with copilot here but guessing that's just cause it's a placeholder yeah?

Reply:
Correct.